tests/rsa/mod.rs

```diff
@@ -26,6 +26,8 @@ pub struct Claims {
     exp: i64,
 }
 
+// Todo: These no longer apply because `verify` does not exist, would probably need to convert it to test the factory for getting signers and verifiers. But I would rather this not be part of the public facing API.
```

Comment:
is it still a todo?

Reply:
Ah true, missed that one. `verify` does indeed exist again.